SYCL/ESIMD/README.md

```diff
@@ -3,3 +3,6 @@ This directory contains ESIMD tests which are run on Intel GPU device only.
 Some of them can run on host device too, but in general it is not always
 possible as some of ESIMD APIs (e.g. memory access via accessors) is not
 implemented for the host device.
+
+macro `ESIMD_TESTS_FULL_COVERAGE` (default mode is `OFF`)
+Enable full coverage with extensive tests.
```

Comment:
```suggestion
Tests within this directory has additional preprocessor definitions available.

`ESIMD_TESTS_FULL_COVERAGE` (default: not defined)\
Enable extended type coverage and testing logic with significantly increased
compilation and execution time. Extensive tests could be run manually if needed,
but do not affect CI performance for the basic functionality
```

Reply:
Applied in this commit: https://github.com/intel/llvm-test-suite/pull/832/commits/acd251de6ae4c78f41534d6c2c1e44f9ced46f9c